beacon-chain/core/peerdas/helpers.go

```diff
@@ -195,6 +196,117 @@ func DataColumnSidecars(signedBlock interfaces.ReadOnlySignedBeaconBlock, blobs
 	return sidecars, nil
 }
 
+// Blobs reconstructs  blobs from data column sidecars.
+// This is the reciprocal function of DataColumnSidecars.
+// `dataColumnsSidecar` must contain at least the first half of `fieldparams.NumberOfColumns` columns,
+// else the function will error. (It can contain more columns, but they be simply ignored.)
+func Blobs(dataColumnsSidecar []*ethpb.DataColumnSidecar) ([]*blocks.VerifiedROBlob, error) {
+	if len(dataColumnsSidecar) == 0 {
+		return nil, nil
+	}
```

Comment:
Should this not return an error or some log as to why the returned `[]*blocks.VerifiedROBlob`  is `nil`?

Reply:
Hum, anyway this behaviour is not consistant at all.
- Giving `0` dataColumnSidecar will return `nil, nil`
- Giving `>0` (but not enough columns will return an error)

--> The `0` case should return an error as well.

Reply:
Fixed in bfa346989423fd960d66d755e0632d3de6881521.